lib/errors.ts

```diff
@@ -180,6 +180,7 @@ const messages: {
 // related issue https://github.com/balena-io/balena-sdk/issues/1025
 // related issue https://github.com/balena-io/balena-cli/issues/2126
 const EXPECTED_ERROR_REGEXES = [
+	/^--detached= cannot also be provided when using --release-tag=$/, // push.ts
```

Comment:
I'd suggest changing to something like:
`\ cannot also be provided when using -\`
As then we will handle all cases where oclif `exlude` rules are broken.